packages/itwinui-css/src/breadcrumbs/breadcrumbs.scss

```diff
@@ -38,28 +38,27 @@
   white-space: nowrap;
   text-overflow: ellipsis;
   line-height: var(--iui-component-height);
+  border: none;
```

Comment:
Not sure why the `border: none` was added, but I think it helps in forced-colors mode, even if it's not visible in regular themes.

Also see https://github.com/iTwin/iTwinUI/pull/1919#r1586559325

Reply:
Did it so test images didn't change, but I removed it and approved the new images. https://github.com/iTwin/iTwinUI/pull/1919#discussion_r1586629845